runtime/js/11_workers.js

```diff
@@ -119,7 +114,7 @@ class Worker extends EventTarget {
       }
     }
 
-    this.#name = name;
+    this.#name = name || "";
```

Comment:
Why this change?

Reply:
There are some unrelated changes that got included during merge. I'll revert them.